packages/paste-core/components/toggle/src/ToggleHelpText.tsx

```diff
@@ -0,0 +1,28 @@
+import * as React from 'react';
+import * as PropTypes from 'prop-types';
+import type {ToggleHelpTextProps} from './types';
+import {HelpText} from '@twilio-paste/help-text';
+import {Box, safelySpreadBoxProps} from '@twilio-paste/box';
+
+const ToggleHelpText = React.forwardRef<HTMLSpanElement, ToggleHelpTextProps>(
+  ({element = 'TOGGLE_HELP_TEXT', children, helpTextId, ...props}, ref) => {
+    return (
+      // @ts-ignore TODO:no 50px tokens
+      <Box {...safelySpreadBoxProps(props)} as="span" display="block" element={element} marginLeft="50px" ref={ref}>
```

Comment:
Question: could we use a 48px token here?

Reply:
it's a bummer
<img width="468" alt="image" src="https://user-images.githubusercontent.com/75342690/183760228-4ca628a4-ae6c-482c-9e0a-0020f945adcc.png">

Reply:
Uhg that is a bummer!